kie-remote/src/main/java/org/kie/remote/command/EventInsertCommand.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2019 Red Hat
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kie.remote.command;
+
+import java.io.Serializable;
+
+public class EventInsertCommand extends WorkingMemoryActionCommand implements VisitableCommand,
```

Comment:
InsertCommand and EventInsertCommand have different purpose ? The names are too similar